config/main.py

```diff
@@ -782,6 +792,64 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN', vlan_name, vlan)
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
+@vlan.group('dhcp_relay')
+@click.pass_context
+def vlan_dhcp_relay(ctx):
+    pass
+
+@vlan_dhcp_relay.command('add')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('dhcp_relay_ip', metavar='<dhcp_relay_ip>', required=True)
```

Comment:
I think we should rename `dhcp_relay_ip` to `relay_destination_ip` or just `destination_ip`